tool/embedded-binaries.ts

```diff
@@ -39,17 +77,22 @@ export async function getEmbeddedProtocol(
 ): Promise<void> {
   const repo = 'embedded-protocol';
   if (options.release) {
-    const version = await downloadRelease({
+    const latestRelease = await getLatestReleaseInfo(
       repo,
-      outPath,
-      version: options.version,
-      assetUrl: releaseInfo =>
-        `https://github.com/sass/${repo}/archive` +
-        `/${releaseInfo.name.replace(' ', '-')}` +
-        getArchiveFileExtension(),
-      tags: true,
-    });
-    fs.rename(p.join(outPath, `${repo}-${version}`), p.join(outPath, repo));
+      options.version,
+      true
```

Comment:
Nit: make this a required named arg, since otherwise it's impossible to guess from reading this what `true` means here.